ssh.go

```diff
@@ -80,7 +81,7 @@ var initAuthMethodOnce sync.Once
 var authMethod ssh.AuthMethod
 
 // initAuthMethod initiates SSH authentication method.
-func initAuthMethod() {
+func initAuthMethod(identityFile string) {
```

Comment:
Doesn't this introduce breaking changes? Better to refactor the code into another function with the identity file parameter and have the old one call the new one with ```null``` as argument to retain the default function's behavior.

```golang

func initAuthMethodWithIdentityFile(identityFile *string) {
    if identityFile != nil {
        // .. Load from identity file. Optionally checking the ENV for a decryption key as well.
    } else {
        // ... default behavior. load from agent etc
    }
}

func initAuthMethod(){
    initAuthMethodWithIdentityFile(nil);
}
```